scrapy/extensions/feedexport.py

```diff
@@ -151,7 +151,7 @@ def __init__(self, settings):
         if not self._exporter_supported(self.format):
             raise NotConfigured
         self.store_empty = settings.getbool('FEED_STORE_EMPTY')
-        self.export_fields = settings.getlist('FEED_EXPORT_FIELDS')
+        self.export_fields = settings.getlist('FEED_EXPORT_FIELDS') or None
```

Comment:
Oh, I was just following through @nramirezuy's comment, personally don't think it matters too much to have that option. If disabling feed export fields wasn't an option before we could merge the pr as it is I think.